gradle/libs.versions.toml

```diff
@@ -11,6 +11,7 @@ guava = "33.1.0-jre"
 java = "11"
 junit = "5.10.2"
 kotlin = "1.9.25"
+kotlineditor = "0.6" # Cannot be called kotlin-editor as it causes `libs.versions.kotlin.get()` to fail
```

Comment:
Note I published two new versions yesterday, so this is now up to 0.8.

Reply:
- [ ] Bump this after [this](https://github.com/cashapp/kotlin-editor/pull/17) is merged

Reply:
Done 52425f7dba261e6b47928ef0bf5b974e50bcbfd8